app/models/cake.rb

```diff
@@ -0,0 +1,9 @@
+class Cake
+  def sweet?
+    true
+  end
```

Comment:
⚠️ **[undercover]** <Untested instance method> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Instance method `sweet?` is missing coverage for line 3 (node coverage: 0.0)
<!-- __reviewdog__:ChA4YWM0M2UzZWFjYmMwYjFiEgp1bmRlcmNvdmVy -->